### DISCUSSION TITLE
Accessibility Violation in Storybook

### DISCUSSION THREAD

**Discussion by mattwendzina** [adobe/react-spectrum#4046] [Q&A]
I'm getting a storybook accessibility violation on a replicated id. I've checked my code to look for anything obvious but can't see anything, the same issue seems to be in this example repo from the docs too:
https://codesandbox.io/s/reverent-faraday-5nwk87?file=/src/DateField.js

Relating it to this repo you can see from the screenshot that the DateField container and then an inner `<div>` both have the same Id - `react-aria7321661300-3`.

![Screenshot](https://user-images.githubusercontent.com/36510083/218254041-678fdafd-d7ed-489f-9d97-29d6d22bd818.jpg)

The accessibility violation says that every id attribute should be unique. 

Is there a reason why this Id is getting replicated and is there a way to avoid this happening?

Thanks

**Comment by devongovett** [comment]
#3969
